cmd/helm/status.go

```diff
@@ -124,6 +127,10 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 		fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
 	}
 
+	if s.showResources && len(s.release.Info.Resources) > 0 {
+		fmt.Fprintf(out, "RESOURCES:\n%s\n", s.release.Info.Resources)
+	}
```

Comment:
This is sending a text blob with the status rather than a JSON structure of the status. kubectl will provide a JSON object with the status when outputting json (or YAML when outputting YAML). Would it be possible to have a JSON output for the status instead of the text block reported for table output?

This would, I think, mean moving the printer usage out of `Get` and to the place it's printed.

Reply:
@mattfarina While testing for this comment I realized helm status command output with this change is same as what was before for json and yaml flags. json and yaml output already prints manifest which is a blob. This PR did make any changes there. Do you recommend updating the existing command?

Reply:
@mattfarina while checking further I realized json and yaml outputs for status command already print manifest which is the is the list of release resources. Do you think it will be a better implementation to remove resources from these output formats and keep it only for normal output mode? What is mean is Resources will be shown only when helm status command is executed without any output formats. json or yaml format will not show helm status and continue to behave the way if it is today

Reply:
@mattfarina I really want to merge this PR in release 3.10 and need a little input and help for this comment. At this point I wonder if we can leave this as is as this does not break the json functionality. Json and Yaml commands already print manifest so resources will just be something additional. What are your thoughts on this? @bridgetkromhout  FYI on the resources printing feature for helm 3.10